Formula/pandoc.rb

```diff
@@ -22,11 +22,6 @@ class Pandoc < Formula
   depends_on "gmp"
```

Comment:
Is this `dependency` really required here? Pandoc builds and works fine without `depends_on "gmp"`.